### DISCUSSION TITLE
Is it possible for terminal tabs to be transferred between browsers?

### DISCUSSION THREAD

**Discussion by kzvi** [coder/code-server#4975] [Q&A]
It seems that if I access code-server from one computer, open a terminal tab, then access it from another computer, the terminal tab doesn't show up on the other computer (whereas text editor tabs do.) Is this a bug or just not supported?

**Comment by jsjoeio** [comment]
Hmm...I don't know if that's supported 🤔 Does it work in vanilla VS Code/Codespaces?

**Reply by kzvi** [reply]
What I mean is that if I open a terminal on my laptop and then shut my laptop down, then go to the same web address on a different computer I can't see any of the open terminals. I can't test this with regular VS code because it is a desktop application.

**Reply by jsjoeio** [reply]
@kzvi can you test this with Codespaces?

**Reply by kzvi** [reply]
No, I don't have access to Codespaces unfortunately.

**Reply by lg-nineoneninethreenine** [reply]
According to my opinion, I think you will have to migrate browser caches, since those must be storing the session details and terminal logs. I don't think this happens with files since code-server kinda has an auto save mode, so hakuna matata in that. Maybe if you could migrate the caches, you could be in luck.

**Reply by jsjoeio** [reply]
Can you reproduce this across browsers? i.e. if you open it in Chrome and then in Firefox, does the same issue happen?

**Reply by kzvi** [reply]
Yes, I can reproduce it that way.

**Reply by jsjoeio** [reply]
Awesome! Next thing you'll want to do is see if this reproduces in VS Code Web. Here are [guidelines](https://github.com/microsoft/vscode/wiki/How-to-Contribute).

If it **does not** happen there, then it's a bug in code-server. If it does, then it's a bug in VS Code.

**Reply by kzvi** [reply]
what do you mean by "vs code web"? did you link the wrong page?

**Reply by jsjoeio** [reply]
VS Code Web is VS Code but that runs in the web (its what code-server uses under the hood). It's also what runs Codespaces.

1. Clone `microsoft/vscode``
2. Follow the "Code for the Web"instructions

<img width="937" alt="image" src="https://user-images.githubusercontent.com/3806031/158266277-a514bda1-9e30-43df-a783-3ee0d8e9f841.png">
